src/config/index.ts

```diff
@@ -99,7 +99,7 @@ const configMap = {
     appConfig: mozambiqueConfig,
     rawLayers: mozambiqueRawLayers,
     rawTables: mozambiqueRawTables,
-    defaultBoundariesFile: `${DEFAULT_BOUNDARIES_FOLDER}/moz_admin_boundaries.json`,
```

Comment:
Let's use this opportunity to use to the simplified config setup for mozambique

Reply:
@ericboucher - done